#!/bin/sh
source="$(find /usr/lib/python*.*/site-packages/dsdrv -name controllers.py)"
cp -a "${source}" "${source}.backup-ds4"
patch "${source}" << 'EOF'
+++ ${source}
--- controllers.py
@@ -45,6 +45,7 @@
 products = {
     "09cc": controllers.DualShock4,
     "054c": controllers.DualShock4,
+    "05c4": controllers.DualShock4,
     "0ce6": controllers.DualSense
 }

EOF

echo >&2 "dsdrv-cemuhook was patched for PS4 controllers."
